src/main/scala/dpla/ingestion3/preMappingReports/Shredder.scala

```diff
@@ -0,0 +1,35 @@
+package dpla.ingestion3.premappingreports
```

Comment:
IntelliJ IDEA is telling me that this package name does not match its directory name. (It must be because it's case-sensitive.) It says it could result in problems resolving dependencies.